### PR TITLE
Npm update

### PR DESCRIPTION
**Pull request by jamesbhobbs**
Resolves n/a

**Overall change:** `npm update` then a load of fixes, pinning, some dev dep corrections then the trivial majjor version updates.

**Code changes:**

- `npm update`
- fix prettier errors
- update snapshots
- pin react-router-config because the updated version has breaking changes
- move lighthouse and storybook to dev deps
- pin storybook deps because they have bugs in the latest version
- update all packages to latest major except:
--"@bbc/gel-foundations": "2.0.0",
--"@bbc/psammead-brand": "^3.0.0",
--"@bbc/psammead-styles": "^0.4.1"
--"@bbc/psammead-timestamp": "^0.3.3",
--"core-js": "^3.1.3",
--"speculate": "^2.0.0"

---

- [x] I have assigned myself to this PR and the corresponding issues
~~- [ ] Tests added for new features~~
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Fragment } from 'react';
 import { configure, addDecorator, addParameters } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
-import { create } from '@storybook/theming';
+import { create } from '@storybook/theming'; // eslint-disable-line import/no-extraneous-dependencies
```
Comment (ryanmccombe) [inline .storybook/config.js:3]: If eslint is complaining about importing dev dependencies - it does support config options that will allow these imports, or allow them in specific globs:

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md

**Review by ryanmccombe — APPROVED** [review]

File: .storybook/config.js (hunk 1 above)
Comment (jamesbhobbs) [reply .storybook/config.js:3]: @ryanmccombe great to know, can you open an issue and if you wish mini-PR to implement this across the repo

**Review by jroebu14 — APPROVED** [review]

File: .storybook/config.js (hunk 1 above)
Comment (ryanmccombe) [reply .storybook/config.js:3]: https://github.com/bbc/simorgh/issues/1886

**Review by jamesdonoh — APPROVED** [review]

**Comment by PriyaKR** [comment]
Regression looks good to me.

**Pull request merged**